queue/queue.go

```diff
@@ -147,6 +151,11 @@ func (q *Queue) Run(c *colly.Collector) error {
 					q.finish()
 					continue
 				}
+				randomDelay := time.Duration(0)
+				if q.RandomDelay != 0 {
+					randomDelay = time.Duration(rand.Intn(int(q.RandomDelay)))
```

Comment:
You can call `sleep` in this line: `time.Sleep(time.Duration(rand.Intn(int(q.RandomDelay))))`